src/ReactiveUI/CommandBinding.cs

```diff
@@ -30,7 +33,7 @@ static CommandBinder()
         /// <param name="view">The View</param>
         /// <param name="viewModel">The View model</param>
         /// <param name="controlName">The name of the control on the view</param>
```

Comment:
>  I think it would be worth being consistent...
Agreed and support this. Grep the build log for ` warning CS1573` for hints of inconsistency that also needs cleaning up

![image](https://cloud.githubusercontent.com/assets/127353/22717354/ab19ea68-edee-11e6-91ed-3165387a058c.png)